particles/Tutorial/Kotlin/3_JsonStore/JsonStore.kt

```diff
@@ -13,16 +13,10 @@ class JsonStoreParticle : Particle() {
   }
 
   override fun populateModel(slotName: String, model: Map<String, String>): Map<String, String> {
-    if(res.get() != null) {
-      return model + mapOf(
-        "name" to res.get()!!.name,
-        "age" to res.get()!!.age.toString()
-      )
-    }
     return model + mapOf(
-        "name" to "",
-        "age" to "0"
-      )
+      "name" to res.get()!!.name ?: "",
```

Comment:
hmm... I think this applies the Elvis operator to `name` and `age` instead of `res.get()`. Maybe something like this? 

```
val pers = res.get() ?: PersonDetails("", 0)
...
"name" to pers.name, 
"age" to pers.age.toString()
```

For reference (see the link I sent): the `!!` is not necessary unless the intention is to fail loudly for debugging purposes.